sound/soc/sof/ops.h

```diff
@@ -470,6 +470,14 @@ snd_sof_set_mach_params(const struct snd_soc_acpi_mach *mach,
 		sof_ops(sdev)->set_mach_params(mach, dev);
 }
 
+/* client register */
+static inline void
+snd_sof_client_register(struct snd_sof_dev *sdev)
```

Comment:
isn't this patch-set incomplete with no calls to this function?

Reply:
yes, I've squashed the last 2 patches.